base/promotion.jl

```diff
@@ -223,21 +223,22 @@ minmax(x::Real, y::Real) = minmax(promote(x, y)...)
 _default_type(T::Type) = (@_pure_meta; T)
 
 if isdefined(Core, :Inference)
-    _promote_op(f::ANY, t::ANY) = Core.Inference.return_type(f, t)
+    _return_type(f::ANY, t::ANY) = Core.Inference.return_type(f, t)
 else
-    _promote_op(f::ANY, t::ANY) = Any
+    _return_type(f::ANY, t::ANY) = Any
 end
 
-promote_op(::Any...) = (@_pure_meta; Any)
```

Comment:
This was unintentional. Of course, it would be nice to get rid of this, but that would have to be done elsewhere. I'll put this back for now.